src/source_sampling.cpp

```diff
@@ -221,14 +223,14 @@ std::vector<double> pyne::Sampler::read_bias_pdf(moab::Range ves,
         for (j=0; j<num_e_groups; ++j){
           q_in_group += pdf[i*num_e_groups + j];
         }
-        if (q_in_group > 0) {
+        if (q_in_group > threshold) {
           for (j=0; j<num_e_groups; ++j) {
             bias_pdf[i*num_e_groups + j] =
                 volumes[i]*pdf[i*num_e_groups + j]/q_in_group;
           }
         } else {
           for (j=0; j<num_e_groups; ++j) {
-            bias_pdf[i*num_e_groups + j] = 0.0;
+            bias_pdf[i*num_e_groups + j] = pdf[i*num_e_groups + j];
```

Comment:
This is a choice... I decided not, but it shouldn't really matter anyway.  It would just result in a different biasing, although slightly higher because the volume is ~125.